content/numerical/PolyInterpolatePuntual.h

```diff
@@ -0,0 +1,29 @@
+/**
+ * Author: Iván Renison
+ * Date: 2024-08-30
+ * License: CC0
+ * Source: me
+ * Description: Given for $i \in \{0, 1, ..., n-1\} $ points $(i, y[i])$ computes de value
+ * of the n-1-degree polynomial that passes through them at point $x$.
+ * Time: O(n \log mod)
+ * status: problem tested
```

Comment:
```suggestion
 * Status: problem tested
```
Sin la mayúscula queda en el PDF